src/client/styles/components/Drbb/Drbb.scss

```diff
@@ -49,7 +49,7 @@
   border-radius: 4px;
   font-size: 14px;
   margin: 20px 0;
-  padding: 5% 8%;
+  padding: 5% 7%;
```

Comment:
Completely stylistic and depends on your convention - but - I'd put everything in px because mixing percentages and pixel values could cause ambiguity going forward.

Reply:
I mean really 90% of this just makes me sad you have to write it instead of grabbing it from design system =(

Reply:
I try to consistently use pixels if it should be the same no matter the screen width, usually a measurement that is just vertical, and percentage when it should vary depending on the screen width. 
But sometimes I end up playing around with percentage and find what I like and slack on replacing it with pixels... Will double check